lib/server.js

```diff
@@ -133,6 +151,6 @@ export default function main() {
         });
     } else {
         const server = new S3Server(cluster.worker);
-        server.startup();
+        server.initiateStartup(logger.newRequestLogger());
```

Comment:
No this will initiate a series of requests to backends, that explicitly expect a request logger.